sources/web/datalab/polymer/modules/SettingsManager.ts

```diff
@@ -12,6 +12,10 @@
  * the License.
  */
 
+interface Window {
```

Comment:
Done.
I had it here because I was getting error messages when I was first figuring this out and had it in common.d.ts, but that must have been due to some other problem, because now it's working.